DaS-PC-MPChan/DSCM.vb

```diff
@@ -371,10 +371,9 @@ Public Class DSCM
                 nmbMaxNodes.Enabled = True
                 nmbMaxNodes.BackColor = New Color()
             Else
-                nmbMaxNodes.Enabled = False
```

Comment:
Ah, I see – this is supposed to fix #66. This is not the correct fix for that issue. The actual bug is in line 369. I will try to take a look at this tomorrow.